string id
ServoDynamics[] servo_table